liubai-frontends/liubai-weixin/miniprogram/behaviors/navibar-behavior.ts

```
@@ -0,0 +1,155 @@
+  lifetimes: {
+    attached() {
+      this.calculateHeight()
+    }
+  },
+
+  pageLifetimes: {
+
+    resize() {
+      const _this = this
+      if(this.data.lastResizeTimeout) {
+        clearTimeout(this.data.lastResizeTimeout)
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)